lib/std/crypto/ripemd160.zig

```diff
@@ -0,0 +1,218 @@
+// https://homes.esat.kuleuven.be/~bosselae/ripemd160.html
+
+const std = @import("std");
+
+pub const Ripemd160 = struct {
```

Comment:
RIPEMD is defined for 128, 160, 256 and 320 bit outputs.

So there should be a `Ripemd(comptime out_bits: usize) type` function that accepts the output size, and `Ripemd160` is `Ripemd(160)`. Similar to what was done for, e.g. Blake2.